src/M6Web/Bundle/ElasticsearchBundle/Connection/CurlMultiConnectionDecorator.php

```diff
@@ -62,8 +65,9 @@ public function performRequest($method, $uri, $params = null, $body = null, $opt
      * @param string $uri
      * @param int    $statusCode
      * @param float  $duration
+     * @param int    $took
      */
-    protected function dispatchEvent($method, $uri, $statusCode, $duration)
+    protected function dispatchEvent($method, $uri, $statusCode, $duration, $took = null)
```

Comment:
`$duration` is the execution time in HTTP client level (with network time). `$took` is the query duration in elasticsearch engine.